client/components/editor/structure/TesList.vue

```diff
@@ -83,22 +77,18 @@ export default {
     },
     onHiddenElementDrawer() {
       this.isElementDrawerVisible = false;
-      this.insertPosition = this.lastPosition;
+      this.insertPosition = null;
```

Comment:
Well, this means `isElementDrawerVisible` has esentially become computed property?
```js
computed: {
  isElementDrawerVisible: ({ insertPosition }) => insertPosition !== null
}
```